src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -105,7 +104,7 @@ export class CalciteDropdownItem {
           id={this.dropdownItemId}
           tabindex="0"
           role="menuitem"
-          aria-selected={selected}
+          aria-selected={this.active ? "true" : null}
```

Comment:
I think you can set this to "false" instead of null, right?

Reply:
Applies to other `aria-selected`

Reply:
The examples I usually see of `aria-selected` generally use `"false"` rather than omitting the attribute all together: https://www.w3.org/WAI/PF/aria/states_and_properties#aria-selected

Setting the attribute to null would actually indicate that the element is not selectable at all I think.

Reply:
Roger that 📻 - updated.